test/metabase/api/async_test.clj

```diff
@@ -13,8 +13,9 @@
 
 (expect
   true
-  (contains? (->> ((user->client :rasta) :get 200
-                   (str "x-ray/field/" (id :venues :price)))
-                  :job-id
-                  (call-with-retries :rasta))
-             :features))
+  (let [client (user->client :rasta)
+        job-id (:job-id (client :get 200 (str "x-ray/field/" (id :venues :price))))
+        _      (result! job-id)]
+    (-> (client :get 200 (str "async/" job-id))
```

Comment:
Since you're not using the `let` clause for anything else here doesn't it make more sense to avoid the `_` binding and just do

```clojure
(let [client (user->client :rasta)
      job-id (:job-id (client :get 200 (str "x-ray/field/" (id :venues :price))))]
  (result! job-id)
  (-> (client :get 200 (str "async/" job-id))
      :result
      (contains? :features)))
```

?

Reply:
Probably. I picked up this habit to mark where things need to be strictly sequential, but since it's not part of our coding standard it just confuses. Will change.